### ISSUE TITLE
[tool] Documentation for building RiscV toolchain needs to be updated

### ISSUE THREAD

**Issue by dbeitel-opentitan** [lowRISC/opentitan#10900]
This page:  https://docs.opentitan.org/doc/ug/install_instructions/#software-development

Describes how to install/build tools for OpenTitan.  The section that describes how to build RiscV gcc toolchain needs to be updated to build a particular version: gcc 9.2.0.  Not sure the exact tag, but I used this one:

% git clone -b rvv-0.9.x --single-branch --recursive https://github.com/riscv/riscv-gnu-toolchain

**Comment by mundaym** [comment]
cc @luismarques

We've pulled the instructions out for now. They had become quite stale and it was a bit strange that we provided instructions for building GCC but not clang (the default compiler in CI).

We could perhaps add in some additional pointers for people while still avoiding providing step by step build instructions. Is there any information that would have been particularly useful to you @dbeitel-opentitan? I'm guessing you are doing the build because you need binaries for macOS?

**Comment by dbeitel-opentitan** [comment]
Yes, I built the compiler to support MacOS.  The instructions were actually very useful, but the only thing missing was the version of the prebuilt package.  If the clang build is a standard build, we should point to common build instructions for the toolchain and the version we built for.

**Comment by dbeitel-opentitan** [comment]
Is it possible for me to get the build instructions for the CLANG toolchain?

**Comment by luismarques** [comment]
> Is it possible for me to get the build instructions for the CLANG toolchain?

Check the [script to build the Clang toolchain](https://github.com/lowRISC/lowrisc-toolchains/blob/master/build-clang-with-args.sh) and the [CI parent script that calls it](https://github.com/lowRISC/lowrisc-toolchains/blob/master/azure-pipelines.yml). While that isn't documentation it should be fairly readable. Let me know if there's anything that isn't clear.
